data/data/gcp/network/lb-public.tf

```diff
@@ -1,25 +1,33 @@
 resource "google_compute_address" "cluster_public_ip" {
+  count = var.public_endpoints ? 1 : 0
```

Comment:
i think we are missing firewall update to make 6443 only accessible to the network https://github.com/openshift/installer/blob/123f41e4dacf26e46dd0660f819841f5d14cb315/data/data/gcp/network/firewall.tf#L1-L13

Reply:
Done. https://github.com/openshift/installer/pull/2522/files#diff-2767d2eaec8456c01ec00ca168a98407L11